rakelib/schemas.rake

```diff
@@ -129,45 +129,21 @@ namespace :schemas do
   desc 'Generate inventory.yaml JSON schema'
   task :inventory do
```

Comment:
This task is greatly simplified now since we have `Bolt::Inventory.schema`. We can likely do something similar for the other schemas, but I'm leaving that for another PR.